spec/controllers/spree/amazon_controller_spec.rb

```diff
@@ -133,41 +133,43 @@
   end
 
   describe "POST #complete" do
+    let(:order) { create(:order_with_line_items, state: 'confirm') }
```

Comment:
@manmartinez updated.  I can sympathize with that perspective on testing, but personally I wouldn't recommend being too strict about it for this repo.  That's not a Solidus convention and imo it's not worth the effort to fight conventions like that as part of a larger community like this when you're accepting PRs from other members of the community.  But in any case I've gone ahead and made the change.